docs/getting_started.md

```diff
@@ -1,64 +1,53 @@
-work in a fast interactive loop: you write some code, save it, review the
-output, write some more, and so on, until you're happy with the results. The
-goal is to use Streamlit to create an interactive app for your data or model
-and along the way to use Streamlit to review, debug, perfect, and share your
-code.
+The easiest way to learn Streamlit is to try things out yourself! As
+you work through this getting started guide, be sure to run each line of code.
+Each time you make a change to your script and save, the Streamlit UI will
+ask if you'd like to rerun the app and view the changes. This workflow allows for
+working in a fast, interactive loop: write some code, save it, review the
+output, write some more, and so on until you're happy with the results.
 
-Use the links below to jump to a specific section:
+## Part 1: Installing Streamlit in a virtual environment
 
-```eval_rst
-.. contents::
-    :local:
-    :depth: 1
-```
-
-## Prerequisites
+Regardless of which package manager you choose ([pipenv](https://pipenv.pypa.io/en/latest/),
+[conda](https://www.anaconda.com/distribution/), [venv](https://docs.python.org/3/library/venv.html), etc.),
+we recommend installing Streamlit in a separate virtual environment for each Streamlit application.
+This ensures that the dependencies pulled in for Streamlit don't impact any other Python projects
+you're working on.
 
-Before you get started, you're going to need a few things:
+These instructions assume you have already installed Python 3.6 or higher on your machine. If you need to download Python,
+please see the [official Python.org website](https://www.python.org/downloads/). Additionally, the [Streamlit wiki](https://github.com/streamlit/streamlit/wiki) provides detailed instructions for [creating Python virtual environments](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment).
 
-- Your favorite IDE or text editor
-- [Python 3.6 or later](https://www.python.org/downloads/)
-- [PIP](https://pip.pypa.io/en/stable/installing/)
-- [Streamlit](index.md) — This one's kind of important, so don't forget to
-  install.
+### Installing Streamlit on macOS and Linux using Pipenv
 
-If you haven't already, take a few minutes to read through [Main
-concepts](main_concepts.md) to understand Streamlit's data flow model.
+The officially-supported environment manager for Streamlit on macOS and Linux is [Pipenv](https://pypi.org/project/pipenv/):
 
-## Set up your virtual environment
+1. Navigate to your project folder: `cd <myproject>`
+2. Install Streamlit in your environment: `pipenv install streamlit`
+   <br />When you run the command above, two files named `Pipfile` and `Pipfile.lock` will appear in `myprojects/`. These files are where your Pipenv environment and its dependencies are declared.
+3. Activate your pipenv environment from the `<myproject>` folder: `pipenv shell`
+4. Test that the installation worked: `streamlit hello`
```

Comment:
IMO the list above looks a little messy / hard to read:

![image](https://user-images.githubusercontent.com/690814/83163435-e20e3100-a0bf-11ea-836d-12d56e56597e.png)

Something like this would be better:

![image](https://user-images.githubusercontent.com/690814/83164070-a2941480-a0c0-11ea-9d17-ae7e4660710c.png)

Reply:
My approach here was to make the page a lot shorter